ibm/mas_devops/roles/suite_db_config/README.md

```diff
@@ -0,0 +1,45 @@
+suite_config
```

Comment:
This should work like the `sls_gencfg` role does (`jdbc_gencfg`), with appropriate changes in the other db2 roles so that there is only one role with the responsibility of generating a jdbccfg template.